miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/ChatMessageBridgeDispatcher.kt

```diff
@@ -0,0 +1,71 @@
+package com.rakuten.tech.mobile.miniapp.js
+
+import com.google.gson.Gson
+import com.rakuten.tech.mobile.miniapp.permission.MiniAppCustomPermissionCache
+import com.rakuten.tech.mobile.miniapp.permission.MiniAppCustomPermissionType
+
+/**
+ * Functionality related to Chat/Messaging. These methods should be implemented by the Host App.
+ */
+@Suppress("TooGenericExceptionCaught")
+abstract class ChatMessageBridgeDispatcher {
```

Comment:
We can make it become `interface` after creating the `ChatMessageBridge` the same `UserInfoBridge`.